packages/utilities/gel-foundations/scripts.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable global-require */
+module.exports = {
+  latin: require('./dist/scripts/latin-and-cyrillic').default,
```

Comment:
It would be good to briefly explain in the Readme why these scripts have been mapped to the these typography settings. Perhaps in a table.

This is so the Readme can be used by those new to supporting multiple scripts, to understand the choices made. It can also be useful for maintaining this package.

Reply:
I agree with Sareh here. I might also be good to link the Dropbox analysis doc assuming it is okay to both be public and is clearly described as a working doc.

Reply:
@pjlee11 Instead of linking to Dropbox, any relevant analysis could be summarised in a document in this package. This increases the visibility and longevity of the documentation - since often dropbox documents can be moved.